index.html

```diff
@@ -116,6 +116,38 @@ <h2>A project for first-time contributions</h2>
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
 
+        <!-- ________ shandamckinney Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Shanda McKinney</p>
+          <p class="contact">
+
+            <i class="fab fa-twitter"></i>
+            <a href="https://twitter.com/shanda_mckinney" target="_blank">@shanda_mckinney/a>
```

Comment:
@shandamckinney check this line, you closing tag for the twitter link is malformed " /a>" please correct and update in this request